luda-editor/luda-editor-client/src/app/types/camera_angle.rs

```diff
@@ -73,17 +73,35 @@ impl CameraAngle {
             height: image_size.height,
         };
 
-        clip(
-            PathBuilder::new().add_rect(&clip_rect),
-            ClipOp::Intersect,
-            namui::image(ImageParam {
-                source: ImageSource::Image(image),
-                xywh: image_xywh,
-                style: ImageStyle {
-                    fit: ImageFit::Fill,
-                    paint_builder: None,
-                },
-            }),
-        )
+        let background = rect(RectParam {
```

Comment:
I found that CameraAngle::render doesn't have background. So I put simple white background here.

Reply:
I think it's better to draw background in [camera_clip_body.rs](https://github.com/NamseEnt/namseent/pull/108/files#diff-c4a608b20e52512e869c2fe29be048a67a3d1efc6ca8dbbe36bdbc2e8bca889b) than in [camera_angle.rs](https://github.com/NamseEnt/namseent/pull/108/files#diff-635a3307a31f5ef5ba090296fa5df5cb72891169da166efa83c282e5e33a57fd).
Because the background is only needed for the clip body.